source/includes/usage-examples/connect-code-examples.php

```diff
@@ -0,0 +1,179 @@
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true, 'tlsCertificateKeyFile' => '/path/to/client.pem'],
+);
+// end-client-cert-client
+
+//  Presents a client certificate to prove identity
+// using connection URI parameters
+// start-client-cert-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsCertificateKeyFile=/path/to/client.pem';
+$client = new MongoDB\Client($uri);
+// end-client-cert-uri
+
+// Specifies the password for a client certificate using client options
+// start-key-file-client
+$client = new MongoDB\Client(
+    'mongodb://<hostname>:<port>/',
+    [
+        'tls' => true,
+        'tlsCertificateKeyFile' => '/path/to/client.pem',
+        'tlsCertificateKeyFilePassword' => '<password>'
+    ],
+);
+// end-key-file-client
+
+// Specifies the password for a client certificate using connection URI parameters
+// start-key-file-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsCertificateKeyFile=/path/to/client.pem&tlsCertificateKeyFilePassword=<password>';
+$client = new MongoDB\Client($uri);
+// end-key-file-uri
+
+// Connects to a TLS-enabled deployment and disables server certificate verification
+// using client options
+// start-insecure-tls-client
+$client = new MongoDB\Client(
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true, 'tlsInsecure' => true],
+);
+// end-insecure-tls-client
+
+// Connects to a TLS-enabled deployment and disables server certificate verification
+// using connection URI parameters
+// start-insecure-tls-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsInsecure=true';
+$client = new MongoDB\Client($uri);
+// end-insecure-tls-uri
+
+// Disables certificate validation using client options
+// start-disable-cert-client
+$client = new MongoDB\Client(
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true, 'tlsAllowInvalidCertificates' => true],
+);
+// end-disable-cert-client
+
+// Disables certificate validation using connection URI parameters
+// start-disable-cert-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsAllowInvalidCertificates=true';
+$client = new MongoDB\Client($uri);
+// end-disable-cert-uri
+
+// Connects to a TLS-enabled deployment and disables hostname verification
+// using client options
+// start-disable-hostname-client
+$client = new MongoDB\Client(
+    'mongodb://<hostname>:<port>/',
+    ['tls' => true, 'tlsAllowInvalidHostnames' => true],
+);
+// end-disable-hostname-client
+
+// Connects to a TLS-enabled deployment and disables hostname verification
+// using connection URI parameters
+// start-disable-hostname-uri
+$uri = 'mongodb://<hostname>:<port>/?tls=true&tlsAllowInvalidHostnames=true';
+$client = new MongoDB\Client($uri);
+// end-disable-hostname-uri
+
+// Connects to a MongoDB deployment and enables the stable API
+// start-stable-api
+$driverOptions = ['serverApi' => new MongoDB\Driver\ServerApi('1')];
+$client = new MongoDB\Client(
+    'mongodb://<hostname>:<port>/',
+    [],
+    $driverOptions,
+);
+// end-stable-api
+
+?>
```

Comment:
```suggestion
```

No need for `?>` in PHP source files.

Reply:
I'll add removing those to the cleanup tasks